server/controllers/gh.js

```diff
@@ -58,6 +60,29 @@ function getIssues(req, res) {
     });
 }
 
+function getIssuesWithLabels(req, res) {
+    logger.log('getIssues');
+
+    const token = req.user && req.user.accessToken;
+
+    Promise.all([
+        makeIssueRequest(issuesRequestUrl, { token, query: req.query }),
+        makeLabelsRequest(labelsRequestUrl, { token })
```

Comment:
Please, lets leave it just as it is for now, but later, when i will be implementing AJAX requests for sorting and filtering - i will fix it